apis/comms_api/middlewares/logging.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import json
+import time
+from starlette.requests import Request
+from starlette.responses import Response
+from starlette.middleware.base import BaseHTTPMiddleware, RequestResponseEndpoint
+
+logger = logging.getLogger('wazuh-comms-api')
+
+async def log_request(request: Request, response: Response, start_time: time) -> None:
```

Comment:
This function is similar to the one we have right now, but the other one goes much deeper to get information about the user, hash context and some other information that I figured wasn't necessary in this case. That's why I didn't re-use the function.